src/stest/Cargo.lock

```diff
@@ -0,0 +1,323 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
+version = 3
```

Comment:
I have a pending work item to git ignore Cargo.lock, so this is okay to include in the PR.